airbyte-integrations/connectors/source-my-hours/source_my_hours/manifest.yaml

```diff
@@ -0,0 +1,104 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: []
+
+  oauth_authenticator:
+    type: CustomAuthenticator
+    class_name: source_my_hours.auth.MyHoursAuthenticator
+    email: "{{ config['credentials']['email] }}"
+    password: "{{ config['credentials']['password'] }}"
+    refresh_token: ""
+
+  bearer_authenticator:
+    type: BearerAuthenticator
+    api_token: "{{ config['credentials']['access_token'] }}"
```

Comment:
IMHO, remove the custom authenticator and only use the BearerAuthenticator which makes the connector possible to build entirely using the connector builder.

Reply:
Unfortunately, even once we add raw-yaml support for Authenticator, `SelectiveAuthenticator` still won't really be possible to implement in the Builder UI alone because it requires nested fields in the connector spec (i.e. a "oneOf" field in the spec)

It seems like we may need to add support for creating oneOf fields in the spec in general based on [these comments](https://docs.google.com/spreadsheets/d/1VReV77rAa4TjLPFksfPFFTvvA3fsy2Ps7kvnLyq1JEs/edit?disco=AAABHA0q-U0)

Reply:
Though that just gave an idea: maybe we should allow the spec (i.e. User Inputs) to be edited in YAML as well for cases like these?